core/block/export/export.go

```diff
@@ -534,8 +564,8 @@ func (e *export) createProfileFile(spaceID string, wr writer) error {
 	return nil
 }
 
-func (e *export) objectValid(sbType smartblock.SmartBlockType, id string, r *model.ObjectInfo, includeArchived bool, isProtobuf bool) bool {
-	if r.Id == addr.AnytypeProfileId {
+func (e *export) objectValid(sbType smartblock.SmartBlockType, id string, details *types.Struct, includeArchived bool, isProtobuf bool) bool {
```

Comment:
We can accept model.ObjectInfo here except separatind object's details and id

Reply:
Fixed